### ISSUE TITLE
Question regarding attribute naming

### ISSUE THREAD

**Issue by magandrez** [exaring/otelpgx#8]
Hi,

First, thank you for taking your time and effort in writing an implementation for Jack's interface for tracing. As I mentioned in https://github.com/exaring/otelpgx/pull/6#issuecomment-1296977895 I (we at Compensate) would really like to help on supporting it (and adopt it, obviously). I have some questions I would like to clarify after reading and digesting [the spec](https://opentelemetry.io/docs/reference/specification/).

As it can be seen in the tracer implementation snippets below (and similar ones throughout the implementation)

https://github.com/exaring/otelpgx/blob/544c13a25b7a0d23f68acfd72058685e5e396e4f/tracer.go#L75-L80

The implementation uses hard-coded keys for the attributes referenced in the snippet, yet it also imports (and makes use) of the [semconv package](https://pkg.go.dev/go.opentelemetry.io/otel@v1.11.1/semconv/v1.10.0) (fixed at 1.10.0 at the time of writing this). Is there any particular reason why not to use the semconv package for those keys? Would you be willing to extend the use of semconv package for this implementation (providing I implement the change)?

Finally, do you have any plans for the library or do you deem it as "feature complete"?

**Comment by obitech** [comment]
Hey @magandrez, sorry I completely missed [your comment](https://github.com/exaring/otelpgx/pull/6#issuecomment-1296977895)!

>Is there any particular reason why not to use the semconv package for those keys? Would you be willing to extend the use of semconv package for this implementation (providing I implement the change)?

To be honest I wasn't aware that the `semconv` package already provides these values 🙈. I'd be more than happy to accept a PR which substitutes our hardcoded values with those from `semconv`.

> Finally, do you have any plans for the library or do you deem it as "feature complete"?

Right now we use the lib in production and we haven't really experienced any issues with it. However we don't use all pgx features such as the `Batch` operations, so I'm not sure how reliable the tracing is in that regard. Since pgx's tracing API is set in place and not subject to change anytime soon (I think), I see this here as feature complete. Until proven otherwise of course: we then need to add features, fix bugs, etc. 

Long story short: I don't see the need for new features right now, but happy to accept PRs or suggestions. Of course bugs should be fixed with high priority. Other than that, I just want to make sure the lib is reasonably up-to-date with the upstream pgx and otel libs.

**Comment by magandrez** [comment]
Related to the change proposed above (#9)

- pgx/v5 tied to v5.0.1 can be upgraded, after a quick glance at the changes in the latest patch version, but I left it out of that PR for safety.
- I believe that `semconv.DBConnectionStringKey` could also be adopted (it is recommended in the spec), although, the heads up given related to the credentials might grant the need for an an option.

What do you think?

**Comment by obitech** [comment]
@magandrez sure, we can do both!

I'm closing this issue since it has been resolved via #9
